app/components/projects/index_page_header_component.html.erb

```diff
@@ -81,6 +81,28 @@
         end
 
         if query.persisted?
+          if can_publish?
```

Comment:
Maybe it is better to introduce a separate `can_delete?` and wrap deletion menu with it instead of `query.persisted?` and put publish menu items outside as `can_publish?` already checks for `query.persisted?`

Reply:
The entire menu items around sharing will be removed when we introduce [the sharing modal](https://community.openproject.org/work_packages/55163/activity). So this is only temporary and will be refactored in the end

Reply:
I added a TODO in the code to mark that

Reply:
It would still be nice to create `can_delete?` method instead of using `query.persisted?`, but is no related to this PR